src/stub_logger.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+/* eslint-env goodeggs/server-side-test */
+import assert from 'assert';
+
+export default function stubLogger (logger: Object, sandbox: Object) {
```

Comment:
I would rename `sandbox` to `sinon` since the two have near-identical interfaces and this function doesn't particularly care if this is a sandbox or the `sinon` singleton.